.gitignore

```diff
@@ -13,6 +13,7 @@
 src/**/*.css
 
 # misc
+.DS_Store
```

Comment:
this should be configured in global gitignore IMHO. I think you can remove this line (and tell mac users to configure their git :p )

Reply:
It's added by react-create-app by default, I removed it then we decided it cost nothing to leave it there, but I agree with you, this has nothing to do here IMO